common/constants.go

```diff
@@ -51,6 +53,7 @@ const (
 	AstraPrivateCloudType = "private"
 	AstraPrivateCloudName = "private"
 
+	ConnectorNeptuneCapability = "neptuneV1"
```

Comment:
Does this need a flag for 23.10 or it should be fine as it is?

Reply:
I am using this flag conditionally only when Deploy Neptune Feature Flag is set to true